doc/usage.rst

```diff
@@ -97,29 +97,34 @@ directory:
     Entering directory '/home/jdimino/code/dune'
     ...
 
-More precisely, it will choose the outermost ancestor directory containing a
-``dune-workspace`` file as root. For instance if you are in
-``/home/me/code/myproject/src``, then dune will look for all these files in
-order:
-
--  ``/dune-workspace``
--  ``/home/dune-workspace``
--  ``/home/me/dune-workspace``
--  ``/home/me/code/dune-workspace``
--  ``/home/me/code/myproject/dune-workspace``
--  ``/home/me/code/myproject/src/dune-workspace``
-
-The first entry to match in this list will determine the root. In
-practice this means that if you nest your workspaces, dune will
-always use the outermost one.
-
-In addition to determining the root, ``dune`` will read this file
-to setup the configuration of the workspace unless the ``--workspace``
-command line option is used. See the section :ref:`dune-workspace`
-for the syntax of this file.
-
-The ``Entering directory`` message can be suppressed with the
-``--no-print-directory`` command line option (as in GNU make).
+This message can be suppressed with the ``--no-print-directory``
+command line option (as in GNU make).
+
+Dune requires at least one ``dune-workspace`` file or ``dune-project``
+file to be present in the current directory or a parent one in order
+to operate. Such a file is used to mark the root of the current
+workspace.
+
+However, they are treated slightly differently. Since there can be
+only a single workspace active at a given time, Dune stops its search
+for the root at the first ``dune-workspace`` file it encounters.
+
+On the other hand, Dune projects are composable and there can be
+multiple projects in a single workspace. For this reason, when Dune
+finds a ``dune-project`` file it will continue searching in parent
+directories in case this project is part of a bigger workspace.
+
+A ``dune-workspace`` file is also a configuration file. ``dune`` will
+read it unless the ``--workspace`` command line option is used.  See
+the section :ref:`dune-workspace` for the syntax of this file. The
+scope of parameters specified in ``dune-project`` file is wider than
```

Comment:
**The parameters' scope** specified in ``dune-project`` file is wider **than
that then** one of parameters specified in ``dune-workspace`` files.

I suspect it's supposed to read "...file is wider than one of the parameters..." so remove "that" and "then"

Reply:
indeed. Reading this sentence again, it feels too complicated. I simplified it to just: "the scope of dune-project files is wider than the scope of dune-workspace files"